### PR TITLE
Several fixes

### PR DESCRIPTION
**Pull request by mathieucarbou**
- no need to use system props anymore: kit path, license path, #stripe, #server, #offheaps, #dataroots, reconnect window are all backup into ${user.home}/.tinypounder/config.properties
- fixing the duplicate EOL in teh console
- fixing mainConsole not focusing at the end of a command

### CODE REVIEW

File: src/main/java/org/terracotta/tinypounder/KitAwareClassLoaderDelegator.java [hunk 1]
```diff
@@ -58,14 +65,17 @@ public KitAwareClassLoaderDelegator() {
...
       clientLibJarStream = Files.walk(Paths.get(kitPath, "/client/lib"), 4, FileVisitOption.FOLLOW_LINKS)
           .filter(path -> path.toFile().getAbsolutePath().endsWith("jar"));
     } catch (IOException e) {
       // it's possible that the client libs are elsewhere...
-      clientLibJarStream = Files.walk(Paths.get(kitPath, "../common/lib"), 4, FileVisitOption.FOLLOW_LINKS)
-          .filter(path -> path.toFile().getAbsolutePath().endsWith("-client.jar"));
+      try {
+        clientLibJarStream = Files.walk(Paths.get(kitPath, "../common/lib"), 4, FileVisitOption.FOLLOW_LINKS)
+            .filter(path -> path.toFile().getAbsolutePath().endsWith("-client.jar"));
+      } catch (IOException ignored) {
```
Comment (anthonydahanne) [inline src/main/java/org/terracotta/tinypounder/KitAwareClassLoaderDelegator.java]: I'm not sure I agree with this...
if there's an exception here, it means no client libs were found either under client/lib nor ../common/lib ; that means it's not a kit, there's something missing.

File: src/main/java/org/terracotta/tinypounder/Settings.java [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    if (reconnectWindow < 5) {
+      reconnectWindow = Integer.parseInt(properties.getProperty("reconnectWindow", "0"));
+      if (reconnectWindow < 5) {
+        reconnectWindow = 120;
+      }
+    }
+  }
+
+  @PreDestroy
+  public void save() {
```
Comment (anthonydahanne) [inline src/main/java/org/terracotta/tinypounder/Settings.java]: is it going to be called when the user "nicely" shutdowns the pounder AND when he / she kills it with ctrl +C ?

File: src/main/java/org/terracotta/tinypounder/TinypounderApplication.java [hunk 3]
```diff
@@ -28,26 +27,10 @@
...
   }
 
-  @Bean
-  public KitAwareClassLoaderDelegator initializeKitAwareClassLoader() {
-    KitAwareClassLoaderDelegator kitAwareClassLoaderDelegator = new KitAwareClassLoaderDelegator();
-    if (kitPath != null && kitPath.length() > 0) {
-      try {
-        kitAwareClassLoaderDelegator.setKitPathAndUpdate(kitPath);
-      } catch (Exception e) {
-        new RuntimeException("Please make sure the kitPath is properly set, current value is : " + kitPath, e);
```
Comment (anthonydahanne) [inline src/main/java/org/terracotta/tinypounder/TinypounderApplication.java]: hum... people should know when the kit is not functional ...

File: src/main/java/org/terracotta/tinypounder/Settings.java (hunk 2 above)
Comment (mathieucarbou) [inline src/main/java/org/terracotta/tinypounder/Settings.java]: yes: ctrl+c == nice shutdown. Just a kill -9 will prevent the save

**Pull request merged**